orangecontrib/spectroscopy/io/neaspec.py

```diff
@@ -310,4 +313,32 @@ def handle_data(self, data):
 
     def _gsf_reader(self, path):
         X, _, _ = reader_gsf(path)
-        return np.asarray(X)
\ No newline at end of file
+        return np.asarray(X)
+    
+class SimpleNeaGSF(FileFormat, SpectralFileFormat):
+
+    EXTENSIONS = (".gsf",)
+    DESCRIPTION = 'Single NeaGSF files'
+
+    def read_spectra(self):
+
+        channel_strings = ['M(.?)A', 'M(.?)P', 'O(.?)A', 'O(.?)P', 'Z C', 'Z raw']
+        for pattern in channel_strings:
+            if re.search(pattern, self.filename) is not None:
+                channel_name = re.search(pattern, self.filename)[0]
+
+        if 'P' in channel_name:
+            datatype = "Phase"
+        elif 'A' in channel_name:
+            datatype = "Amplitude"
+        elif 'Z' in channel_name:
+            datatype = "Topography"
+        else:
+            datatype = "None"
+
+        X, XRr, YRr = reader_gsf(self.filename)
+        features, final_data, meta_data = _spectra_from_image(X, np.array([1]), XRr, YRr)
+
+        meta_data.attributes["datatype"] = datatype
```

Comment:
datatype is too general. We need a better name for this.

Reply:
I think we should keep it the meta.attribute name as "datatype" or at least I can't come up with a better name for now. The purpose is to be general because in the future we should try to add and use this kind of information with the data. As an example, IR and Raman spectra have some specifics and we could use this information elsewhere, like automatically setting the axes labels on the plots. 

In this particular case I was thinking to change it to "neadatatype" but this would not allow us to use this particular key with non-neaspec instruments. Then I thought "snomdatatype" but neaspec instruments can measure other stuff, not only snom... So I think a general name is the best here and gives us the flexibility for the future.

@markotoplak, what do you think with these arguments is it OK to merge?

Reply:
I was also not able to work out a better name. But do we at least have a replacement for "data", that could imply that we have amplitude and phase in question? I can't think of a better word. Maybe `signal`? No idea.

Or maybe some prefix, like `spectra.datatype` or something better?

Reply:
**I think "signal_type" is a good idea!**

The main point is to expand beyond the amplitude/phase box so that we can use the same meta.attribute later for other things like Raman or Mirage (which started to have many types of data coming out from it).

Then we could add later another meta.attribute for "method" that would distinguish between instrumental differences such as Raman vs. IR vs. SNOM but this is a discussion for later.

Reply:
Ok, then I change this to `signal_type` and take care that image preprocessor search for the new name.

Reply:
@ngergihun, wait. :) I would also prefer some prefix, like `spectroscopy.` or something smarter, so like `spectroscopy.signal_type` or maybe `measurement.signal_type`. @borondics, @ngergihun, if you think something similar applies, good, otherwise, just `signal_type` is good enough.

Reply:
@markotoplak okay, let's use `measurement.signaltype` :D :D